setup.py

```diff
@@ -31,15 +31,8 @@
         'Images':        ["opencv-python>=4.0"],
         'Rhino':         ["rhino3dm>=0.14"],
     },
-    # ext_modules=cythonize("splipy/basis_eval.pyx"),
-    ext_modules=cythonize([
-        Extension(
-            'splipy.basis_eval',
-            ['splipy/basis_eval.pyx'],
-            include_dirs=[np.get_include()],
-        )
-    ]),
-    # include_dirs=[np.get_include()],
+    ext_modules=cythonize("splipy/basis_eval.pyx"),
+    include_dirs=[np.get_include()],
```

Comment:
Was this fixed or not?

Reply:
No this was a mistake. It should not be included.

Reply:
Revoked `setup.py` changes